apps/website/components/cards/Card.tsx

```diff
@@ -0,0 +1,63 @@
+import { classed } from "@tw-classed/react"
+
+const CardBase = classed.div(
+    "rounded-lg break-all overflow-hidden duration-300 ease-in-out cursor-pointer",
```

Comment:
```suggestion
    "rounded-lg overflow-hidden duration-300 ease-in-out cursor-pointer",
```

I think it will be better to remove this property so that the text does not split incorrectly.